o2dpg.sh

```diff
@@ -1,6 +1,6 @@
 package: O2DPG
 version: "%(tag_basename)s"
-tag: master
+tag: 9ba38e202024c2d844bbafca191331155a685be1
```

Comment:
I think it's enough to drop "version".

Reply:
ok, but I thought our conclusion was to retag daily?
So we would not drop the `version`?